src/builds.ts

```diff
@@ -0,0 +1,279 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
If you wanted, you could just use @vscode/test-electron for this. It has a `downloadAndUnzipVSCode` which does what you do here, with a little more resilience (retries and timeouts). May just need to add some functionality to target specific commits if you eventually need it.

Reply:
That's great to know. Will adopt it as next step. May I know if it can support `x-vscode-released: false` header so that we can also get unreleased builds?

Reply:
Lets track this here - https://github.com/microsoft/vscode-perf/issues/10